### PR TITLE
New v2

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem

- New v2 branch for testing purpose to not mess up with the one from ericg

Replace #204

## Solution

- *And how do you fix that problem*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    type = "user"
+
+[resources]
+    [resources.sources]
+        [resources.sources.api]
+        url = "https://dev.funkwhale.audio/funkwhale/funkwhale/-/jobs/artifacts/1.2.10/download?job=build_api"
+        sha256 = "fc6f54d37993f74e057d1a59438e21b68a8cff1a7f9438415459ede7cf7f09d0"
+        in_subdir = true
+        extract   = true
+        format = "zip"
```
Comment (ericgaspar) [inline manifest.toml:51]: not sure you need it...

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+        [resources.sources.front]
+        url = "https://dev.funkwhale.audio/funkwhale/funkwhale/builds/artifacts/1.2.10/download?job=build_front"
+        sha256 = "bcf0ba380295be58dfcfe4d88605cfce357859adf623c8236fb1182095c38bfd"
+        in_subdir = true
+        extract   = true
+        format = "zip"
```
Comment (ericgaspar) [inline manifest.toml:58]: same

File: scripts/change_url [hunk 3]
```diff
@@ -142,14 +75,7 @@ ynh_systemd_action --service_name="$app-worker" --action="start" --log_path="/va
...
 # Create a dedicated Fail2Ban config
-ynh_add_fail2ban_config --logpath="/var/log/nginx/$new_domain-access.log" --failregex="<HOST>.* \"POST /api/v1/token/ HTTP/1.1\" 400 68.*$" --max_retry=5
-
-#=================================================
-# RELOAD NGINX
-#=================================================
-ynh_script_progression --message="Reloading NGINX web server..." --weight=1
-
-ynh_systemd_action --service_name=nginx --action=reload
+#REMOVEME? ynh_add_fail2ban_config --logpath="/var/log/nginx/$new_domain-access.log" --failregex="<HOST>.* \"POST /api/v1/token/ HTTP/1.1\" 400 68.*$" --max_retry=5
```
Comment (ericgaspar) [inline scripts/change_url:78]: remove / not remove?

File: scripts/upgrade [hunk 4]
```diff
@@ -59,95 +26,63 @@ ynh_systemd_action --action="stop" --service_name="${app}-beat" --log_path="syst
...
-	ynh_secure_remove --file="$final_path/import"
-	ynh_secure_remove --file="$final_path/code/data/music"
+#REMOVEME? 	ynh_secure_remove --file="$install_dir/code"
+#REMOVEME? 	ynh_secure_remove --file="$install_dir/media"
+#REMOVEME? 	ynh_secure_remove --file="$install_dir/code/data/media"
+#REMOVEME? 	ynh_secure_remove --file="$install_dir/import"
+#REMOVEME? 	ynh_secure_remove --file="$install_dir/code/data/music"
 
-	ynh_app_setting_set --app=$app --key=datadir --value=$datadir
+#REMOVEME? 	ynh_app_setting_set --app=$app --key=data_dir --value=$data_dir
```
Comment (ericgaspar) [inline scripts/upgrade:77]: to be removed

**Comment by Thovi98** [comment]
Thanks for your suggestions @ericgaspar!

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
Just tested, everything works as expected!

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**